docs/preview/02-Features/powershell/azure-security.md

```diff
@@ -58,8 +58,7 @@ When you want to make use of the REST-API's made available to manage Azure Resou
 
 | Parameter               | Mandatory | Description                                                                                       |
 | ----------------------- | --------- | ------------------------------------------------------------------------------------------------- |
-| `AssignGlobalVariables` | no        | Switch - default value: false                                                                     |
-|                         |           | When you want the global variables `access_token` and `subscriptionId` assigned for easy access.  |
+| `AssignGlobalVariables` | no        | Indicator (switch) whether you want the global variables `access_token` and `subscriptionId` assigned for easy access.  |
```

Comment:
We should mention that the default is `false`.

Reply:
Something like this?
Indicator (switch - default value: false) whether you want the global variables `access_token` and `subscriptionId` assigned for easy access.

Reply:
Yes, that's ok. 👍